server/amp/handlers/infinite-scroll.js

```diff
@@ -19,7 +19,7 @@ async function storyPageInfiniteScrollHandler(
     client,
```

Comment:
Except for bumping amplib in package.json, all other changes here aren't exactly needed for this PR. 
`itemsTaken has to match with itemsToTake in ampStoryPageHandler` this can cause issues in future, so fixing it